NDB.Covid19/NDB.Covid19/Models/SelfDiagnosisSubmissionDTO.cs

```diff
@@ -29,6 +32,7 @@ public SelfDiagnosisSubmissionDTO()
             Platform = ServiceLocator.Current.GetInstance<IDeviceInfo>().Platform.ToString();
             Regions = Conf.SUPPORTED_REGIONS.ToList();
             IsSharingAllowed = LocalPreferencesHelper.AreCountryConsentsGiven;
+            ReportType = LocalPreferencesHelper.IsReportingSelfTest ? (int)TemporaryExposureKey.Types.ReportType.SelfReport : (int)TemporaryExposureKey.Types.ReportType.ConfirmedTest;
```

Comment:
Please do it in a cleaner way: `(int)ReportType.SelfReport`

Reply:
This results in a name conflict with the member from the dto. Updated to reference the type directly.

Reply:
Got it